ScoutSuite/output/data/inc-scoutsuite/scoutsuite.js

```diff
@@ -1039,10 +1038,6 @@ function get_resource_path(path) {
         resource_path_array = resource_path.split('.');
         last_value = resource_path_array.pop();
         resource_path = 'services.' + resource_path_array.join('.');
-        // Fix for issue #79
-        if (last_value == '<root_account>') {
-            resource_path += '.' + last_value;
-        };
```

Comment:
This page does not exist anymore. Clicking on root-account specific rules will redirect to the credential-reports dashboard with the <root-account> focused.